dev/com.ibm.ws.jaxrs.2.0.common/bnd.bnd

```diff
@@ -40,7 +40,8 @@ Export-Package: \
 	com.ibm.websphere.jaxrs20.multipart;version=1.1.0,\
 	com.ibm.ws.jaxrs20.clientconfig,\
 	org.apache.cxf.feature,\
-	org.apache.cxf.jaxrs.client;version=3.1.18
+	org.apache.cxf.jaxrs.client;version=3.1.18,\
+	com.ibm.ws.cxf.exceptions;version=3.1.18
```

Comment:
It is only used internal to this bundle.  Does it really need to be exported?

Reply:
technically no, but I was mirroring 3.2. I can remove it.

Reply:
I tried removing and got unresolved import errors. I need to keep exporting it.